src/projects/projects-page.html

```diff
@@ -17,76 +17,58 @@
     <style>
       :host {
         flex: 1;
-        /*overflow: hidden;*/
         display: flex;
-        -webkit-transition: all 500ms ease;
-        -moz-transition: all 500ms ease;
-        -ms-transition: all 500ms ease;
-        -o-transition: all 500ms ease;
-        transition: all 500ms ease;
       }
       [hidden] {
         display: none !important;
       }
-      aside {
+      .collapsable {
```

Comment:
It was, but strangely it added some white-space on top of each list that I could not remove using padding or margin. Changing to `div` solved that problem